src/managed/Microsoft.Extensions.DependencyModel/DependencyContextJsonReader.Netstandard2_0.cs

```diff
@@ -0,0 +1,841 @@
+                    written += lastRead;
+                } while (lastRead > 0);
+
+                return new ArraySegment<byte>(rented, 0, written);
+            }
+            catch
+            {
+                if (rented != null)
+                {
+                    // Holds document content, clear it before returning it.
+                    rented.AsSpan(0, written).Clear();
+                    ArrayPool<byte>.Shared.Return(rented);
+                }
+                throw;
+            }
+        }
+
+        private DependencyContext ReadCore(ReadOnlySpan<byte> jsonData, byte[] rentedBuffer)
+        {
+            var jsonReader = new Utf8JsonReader(jsonData, isFinalBlock: true, state: default);
+
+            jsonReader.ReadStartObject();
+
+            var runtime = string.Empty;
+            var framework = string.Empty;
+            var isPortable = true;
+            string runtimeTargetName = null;
+            string runtimeSignature = null;
+
+            CompilationOptions compilationOptions = null;
+            List<Target> targets = null;
+            Dictionary<string, LibraryStub> libraryStubs = null;
+            List<RuntimeFallbacks> runtimeFallbacks = null;
+
+            while (jsonReader.Read() && jsonReader.TokenType == JsonTokenType.PropertyName)
+            {
+                switch (jsonReader.GetStringValue())
+                {
+                    case DependencyContextStrings.RuntimeTargetPropertyName:
+                        ReadRuntimeTarget(ref jsonReader, out runtimeTargetName, out runtimeSignature);
+                        break;
+                    case DependencyContextStrings.CompilationOptionsPropertName:
+                        compilationOptions = ReadCompilationOptions(ref jsonReader);
+                        break;
+                    case DependencyContextStrings.TargetsPropertyName:
+                        targets = ReadTargets(ref jsonReader);
+                        break;
+                    case DependencyContextStrings.LibrariesPropertyName:
+                        libraryStubs = ReadLibraries(ref jsonReader);
+                        break;
+                    case DependencyContextStrings.RuntimesPropertyName:
+                        runtimeFallbacks = ReadRuntimes(ref jsonReader);
+                        break;
+                    default:
+                        jsonReader.Skip();
+                        break;
+                }
+            }
+
+            if (compilationOptions == null)
+            {
+                compilationOptions = CompilationOptions.Default;
+            }
+
+            Target runtimeTarget = SelectRuntimeTarget(targets, runtimeTargetName);
+            runtimeTargetName = runtimeTarget?.Name;
+
+            if (runtimeTargetName != null)
+            {
+                var seperatorIndex = runtimeTargetName.IndexOf(DependencyContextStrings.VersionSeperator);
```

Comment:
Typo: sepArator

Reply:
Didn't introduce it, so would have to change it in both places. Will fix.